meta-resin-common/recipes-connectivity/dnsmasq/files/dnsmasq.conf.systemd

```diff
@@ -5,4 +5,4 @@ After=resin-net-config.service
 [Service]
 OOMScoreAdjust=-1000
 ExecStart=
-ExecStart=/usr/bin/dnsmasq -x /run/dnsmasq.pid -a 127.0.0.2 -7 /etc/dnsmasq.d/ -r /etc/resolv.dnsmasq -z --servers-file=/run/dnsmasq.servers
+ExecStart=/usr/bin/dnsmasq -x /run/dnsmasq.pid -a "127.0.0.2,10.114.102.1" -7 /etc/dnsmasq.d/ -r /etc/resolv.dnsmasq -z --servers-file=/run/dnsmasq.servers
```

Comment:
The git log could use some rework, it's a bit tangled

Reply:
Reworded